query_server/query/src/metadata.rs

```diff
@@ -166,6 +176,183 @@ impl MetaData for LocalCatalogMeta {
+                    .unwrap();
+
+                let batches = vec![Arc::new(batch)];
+
+                Ok(Output::StreamData(stream_from_batches(batches)))
+            }
+        }
+    }
+
+    fn describe_database(&self, name: &str) -> Result<Output> {
+        match self.engine.get_db_schema(name) {
+            None => Err(MetadataError::DatabaseNotExists {
+                database_name: name.to_string(),
+            }),
+            Some(db_cfg) => {
+                let schema = Arc::new(Schema::new(vec![
+                    Field::new("TTL", DataType::Utf8, false),
+                    Field::new("SHARD", DataType::Utf8, false),
+                    Field::new("VNODE_DURATION", DataType::Utf8, false),
+                    Field::new("REPLICA", DataType::Utf8, false),
+                    Field::new("PRECISION", DataType::Utf8, false),
+                ]));
+
+                let ttl = db_cfg.config.ttl.to_string();
+                let shard = db_cfg.config.shard_num.to_string();
+                let vnode_duration = db_cfg.config.vnode_duration.to_string();
+                let replica = db_cfg.config.replica.to_string();
+                let precision = db_cfg.config.precision.to_string();
+
+                let batch = RecordBatch::try_new(
+                    schema,
+                    vec![
+                        Arc::new(StringArray::from(vec![ttl.as_str()])),
+                        Arc::new(StringArray::from(vec![shard.as_str()])),
+                        Arc::new(StringArray::from(vec![vnode_duration.as_str()])),
+                        Arc::new(StringArray::from(vec![replica.as_str()])),
+                        Arc::new(StringArray::from(vec![precision.as_str()])),
+                    ],
+                )
+                .unwrap();
+
+                let batches = vec![Arc::new(batch)];
+
+                Ok(Output::StreamData(stream_from_batches(batches)))
+            }
+        }
+    }
+
+    fn describe_table(&self, table: &str) -> Result<Output> {
+        let table_name;
+        let database_name;
+
+        if table.contains('.') {
+            (database_name, table_name) = table.split_once('.').unwrap();
+        } else {
+            table_name = table;
+            database_name = self.database_name.as_str();
+        }
+
+        match self
+            .engine
+            .get_table_schema(database_name, table_name)
+            .unwrap()
+        {
+            None => Err(MetadataError::TableNotExists {
+                table_name: table.to_string(),
+            }),
+            Some(table_schema) => {
+                let schema = Arc::new(Schema::new(vec![
+                    Field::new("fieldname", DataType::Utf8, false),
+                    Field::new("type", DataType::Utf8, false),
+                    Field::new("istag", DataType::Boolean, false),
+                    Field::new("compression", DataType::Utf8, false),
```

Comment:
The returned metadata field names need to be unified.

```
                    Field::new("TTL", DataType::Utf8, false),
                    Field::new("SHARD", DataType::Utf8, false),
                    Field::new("VNODE_DURATION", DataType::Utf8, false),
                    Field::new("REPLICA", DataType::Utf8, false),
                    Field::new("PRECISION", DataType::Utf8, false),
```

Reply:
nice hint